docker-compose.yml

```diff
@@ -18,6 +21,7 @@ services:
       - db
     env_file:
       - ./local.env
+      - ./db/certs/test.env
```

Comment:
why not the same path/required logic for local.env here?

Reply:
It's only necessary for files that are git-ignored